src/com/session/SessionHelper.java

```diff
@@ -46,37 +48,65 @@ public static boolean isPresent(String email) {
 
 	}
 
-	public static PostPojo getPostContent() throws IOException {
-
-		org.codehaus.jackson.map.ObjectMapper mapper = new org.codehaus.jackson.map.ObjectMapper();
-		URL url = new URL("http://jsonplaceholder.typicode.com/posts/1");
-
-		HttpURLConnection con = (HttpURLConnection) url.openConnection();
-		con.setRequestMethod("GET");
-		con.connect();
-
-		StringBuilder sb = new StringBuilder();
+	public static PostPojo getPostContent(String requestNumber) throws IOException, JSONException {
 
-		BufferedReader br = new BufferedReader(new InputStreamReader(con.getInputStream()));
-
-		String line;
-		while ((line = br.readLine()) != null) {
-			sb.append(line + "\n");
+		ObjectMapper mapper = new ObjectMapper();
+		PostPojo post = null;
+		JSONObject json = new JSONObject();
+		try {
+			String temp = requestNumber.substring(1);
+			int lastChar = Integer.parseInt(temp);
+			System.out.println(lastChar);
+			if (lastChar > 0 && lastChar < 11) {
+				String actualUrl = "http://jsonplaceholder.typicode.com/posts" + requestNumber;
+				URL url = new URL(actualUrl);
+
+				HttpURLConnection con = (HttpURLConnection) url.openConnection();
+				con.setRequestMethod("GET");
+				con.connect();
+
+				StringBuilder sb = new StringBuilder();
+
+				BufferedReader br = new BufferedReader(new InputStreamReader(con.getInputStream()));
+
+				String line;
+				while ((line = br.readLine()) != null) {
+					sb.append(line + "\n");
+				}
+				br.close();
+
+				json.put("ok", true);
+				json.put("message", "found");
+				String string = sb.toString();
+				String jsonString = json.toString();
+				string=string.concat(jsonString);
+				post = mapper.readValue(string, PostPojo.class);
+				System.out.println(string);
+				//post = mapper.readValue(jsonString, PostPojo.class);
+
+				return post;
+			} else {
+
+				json.put("ok", false);
```

Comment:
this method is meant to return only Post Pojo, not other api response properties like ok, message etc, those needs to be done in servlets , based on pojo is null or not